content/the-road-to-ember-4-0.md

```diff
@@ -0,0 +1,89 @@
+more details.
+
+Scanning this list, the trend is that many rarely used APIs dating back to Ember 1.x patterns are removed.
+
+However the most notable change is the completion of Ember 3.x's browser support policy. Ember 4.0 will support two classes of browsers, evergreen (those on a weeks-long auto-upgrade release cycle) and non-evergreen. This classification system allows us to create a rolling minimum version for evergreen browsers while using a more traditional pinned minimum version for non-evergreen browsers.
+
+**Specifically, the Ember 4.x release policy includes support for Google Chrome, Mozilla Firefox, Microsoft Edge, and Apple Safari on desktop and mobile. It does not include support for any version of Internet Explorer.**
+
+Read more about this change in [the deprecation guide](https://deprecations.emberjs.com/v3.x/#toc_3-0-browser-support-policy) and at [Ember's browser support policy page](https://emberjs.com/browser-support/).
+
+Existing Ember users should note that Ember 3.27 has already removed IE11 from the default target list for production and testing builds.
+
+### Planning your upgrade to 4.0
+
+For each API removed in Ember 4.0 you can find an entry in the [Ember 3.x deprecation guide](https://deprecations.emberjs.com/v3.x/). In many applications, you have probably addressed many of these warnings already.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon as part of your upgrade process. The workflow addon allows you to create a configuration file to silence most deprecations, allowing you to focus on one or a few warnings at a time. If you're working with a large codebase and many contributors, the focused control of deprecations will also allow you to throw when deprecations you've already addressed are re-introduced, preventing backsliding in your upgrade process.
+
+In contrast to prior major releases, we have no plans to offer an
+`ember-3-legacy` package which makes Ember 3.x APIs available for a limited time
+in Ember 4.0. This is based on fairly low use of these packages in the past.
+
+**Don't panic.** We expect many Ember users to be enthusiastic to adopt our 4.0 release, but for conservative consumers we continue to suggest using Ember's Long-Term Support releases. Ember 3.28, the last version of the 3.x cycle, will become an LTS in September 2021. The first 4.x release to be promoted to LTS will be Ember 4.4 around March 2022. This timeline provides a long window for your business to address any remaining 4.0-targeted deprecations if you're on the LTS cycle.
+
+### Contributing to Ember 4.0
+
+Across [Ember.js](https://github.com/emberjs/ember.js), [Ember Data](https://github.com/emberjs/data), and [Ember CLI](https://github.com/ember-cli/ember-cli) repos the final beta cycle of 3.x has already started. The `master` branch of each repo is ready for contribution to remove already deprecated functionality and IE11 compatibility code.
```

Comment:
I wasn't sure sure if the 2nd sentence was a call-to-action (please contribute). Maybe we can make it clearer by writing,

```suggestion
In [Ember.js](https://github.com/emberjs/ember.js), [Ember Data](https://github.com/emberjs/data), and [Ember CLI](https://github.com/ember-cli/ember-cli) repos, the final beta cycle of 3.x has already started. You are welcome to contribute to these repos by removing already deprecated functionality and IE11 compatibility code.
```

?